### DISCUSSION TITLE
聚合翻译和新增同步选项

### DISCUSSION THREAD

**Discussion by heyseere** [immersive-translate/immersive-translate#1041] [Ideas]
### 新功能 | New Feature

1. 聚合翻译（例如：单词数 10 以下的段落使用 Deepl 翻译，长段落使用 ChatGPT 翻译。）
2. 增加 OneDrive 同步

### 为什么您想要添加这些功能 | Why you want to add these features

1. 聚合翻译可以有效加快页面翻译速度
2. 国内网络环境使用 Google Drive 不便

**Comment by keyboard3** [comment]
第一点：不会采用混合翻译模式。聚合翻译 deepl现已针对网页中文章内容部分进行段落合并翻译
第二点：不会增加其他同步方案，你可以使用pro会员的自动同步
